src/components/__tests__/EditVocabulary.test.tsx

```
@@ -0,0 +1,695 @@
+        }
+        if (functionName === 'translate-word') {
+          return Promise.resolve({
+            data: { translation: 'ai-translated' },
+            error: null,
+          });
+        }
+        return Promise.resolve({ data: {}, error: null });
+      });
+  });
+
+  test.only('loads and displays vocabulary data correctly', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)